test/test_support.ts

```diff
@@ -19,8 +19,25 @@ import * as ts from 'typescript';
 import * as cliSupport from '../src/cli_support';
 import * as tsickle from '../src/tsickle';
 
-/** Path to tslib.d.ts; used inside Google for this test suite. */
-const tslibPath: string|null = null;
+/**
+ * Return the 'root dir' under tests, which is the path to the image of
+ * the source tree set up under the hermetic testing environment.
+ */
+function rootDir(): string {
+  const runfiles = process.env['RUNFILES'];
+  assert(runfiles);
+  return path.join(runfiles!, 'tsickle');
+}
+
+/**
+ * Path to tslib.d.ts, relative to the project root.
+ *
+ * The test suite runs the TS compiler, which wants to load tslib.
```

Comment:
I don't understand why we need tslib to begin with - do have test code that has an import of tslib? that does seem odd.